test/tendermint.dynval/snapshot-config.yml

```diff
@@ -1,6 +1,7 @@
 [codechain]
 
 [mining]
+self_nomination_enable = false
```

Comment:
Why did you change it?

Reply:
That's the quick fix for the error 
```
Error: "Error while parsing TOML: Error { inner: ErrorInner { kind: Custom, line: None, col: 0, message: \"missing field `self_nomination_enable`\", key: [\"mining\"] } }"
```
We also can designate default value for the field.